robottelo/test.py

```diff
@@ -151,7 +151,10 @@ def setUpClass(cls):  # noqa
     def tearDownClass(cls):
         super(UITestCase, cls).tearDownClass()
         if cls.session_user is not None:
-            cls.session_user.delete(synchronous=False)
+            try:
+                cls.session_user.delete(synchronous=False)
+            except Exception as e:
```

Comment:
I think we should use a specific exception here because Exception is so generic that can mask other possible failures. This will do what it is supposed to do, but always that I see generic Exception that does not look good to me.